packages/server-wallet/src/wallet/store.ts

```diff
@@ -431,6 +431,11 @@ export class Store {
     await AppBytecode.upsertBytecode(chainNetworkId, appDefinition, bytecode, this.knex);
   }
 
+  async getObjective(objectiveId: string, tx?: TransactionOrKnex): Promise<DBObjective> {
+    tx = tx || this.knex; // todo: make tx required
```

Comment:
Is this TODO now?

Reply:
It was really meant for a later date, when we make the store more consistent.